packages/ag-charts-community/src/chart/tooltip/tooltipContent.ts

```diff
@@ -96,6 +96,10 @@ function dataHtml(label: string | undefined, value: string, inline: boolean) {
 function tooltipRowContentHtml(content: GroupedStructuredContent['items'][0]) {
     let html = '';
 
+    if (content.data?.length && content.data.every((datum) => !datum.value)) {
```

Comment:
Maybe to avoid `0` not being rendered?
```suggestion
    if (content.data?.length && content.data.every((datum) => datum.value != null)) {
```

Reply:
Updated to `datum.value != null && datum.value !== ''` to prevent empty strings but allow if a `0` sneaks in.

Reply:
That logic was inverted 🤦‍♂️, fixed.